lib/jsCompiler.js

```diff
@@ -84,6 +85,21 @@ module.exports = function (app, callback) {
     jsFiles = klawSync(jsPath)
   }
 
+  // read files to ignore from .gitignore
+  fs.readFile('.gitignore', (err, data) => {
+    let split
+    if (err) {
+      logger.error(err)
+    }
+    split = data.toString().split('\n')
+    split.forEach((line) => {
+      // skip blank lines
+      if (line !== '') {
```

Comment:
should account for other kinds of invalid entries too, like comments or whole lines filled with long whitespace